nginx/conf.d/scalelite-ssl.template

```diff
@@ -4,8 +4,11 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
+    location / {
+        return  301 https://$host$request_uri;
+      }
+
 	include conf.d/scalelite/*;
-	include conf.d/scalelite.common;
```

Comment:
Did you mean to remove this?

Reply:
Yeah, I was following @jfederico 's lead on it. The individual file reference was removed in his latest commit.

Reply:
Ah my mistake. I got confused with something else